scripts/install/prowlarr.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+# *Arr Installer for Prowlarr
+# Refactored from existing files by FlyingSausages and others
+# Bakerboy448 2021 for Swizzin
+
+#shellcheck source=sources/functions/utils
+. /etc/swizzin/sources/functions/utils
+
+app_name="prowlarr"
+if [ -z "$PROWLARR_OWNER" ]; then
+    if ! PROWLARR_OWNER="$(swizdb get "$app_name/owner")"; then
+        PROWLARR_OWNER="$(_get_master_username)"
+        echo_info "Setting ${app_name^} owner = $PROWLARR_OWNER"
+        swizdb set "$app_name/owner" "$PROWLARR_OWNER"
+    fi
+else
+    echo_info "Setting ${app_name^} owner = $PROWLARR_OWNER"
+    swizdb set "$app_name/owner" "$PROWLARR_OWNER"
+fi
+
+_install_prowlarr() {
+    user="$PROWLARR_OWNER"
+    app_configdir="/home/$user/.config/${app_name^}"
+    app_port="9696"
+    app_reqs=("curl" "sqlite3")
+    app_servicename="${app_name}"
+    app_servicefile="$app_servicename".service
+    app_dir="/opt/${app_name^}"
+    app_binary="${app_name^}"
+    app_lockname=$app_name
+    app_group="$user"
+
+    apt_install "${app_reqs[@]}"
+
+    if [ ! -d "$app_configdir" ]; then
+        mkdir -p "$app_configdir"
+    fi
+    chown -R "$user":"$app_group" "$app_configdir"
+
+    echo_progress_start "Downloading release archive"
+
+    #ToDo: Update branch
+    urlbase="https://$app_name.servarr.com/v1/update/nightly/updatefile?os=linux&runtime=netcore"
+    case "$(_os_arch)" in
+        "amd64") dlurl="${urlbase}&arch=x64" ;;
+        "armhf") dlurl="${urlbase}&arch=arm" ;;
+        "arm64") dlurl="${urlbase}&arch=arm64" ;;
+        *)
+            echo_error "Arch not supported"
+            exit 1
+            ;;
+    esac
+
+    if ! curl "$dlurl" -L -o "/tmp/$app_name.tar.gz" >> "$log" 2>&1; then
+        echo_error "Download failed, exiting"
+        exit 1
+    fi
+    echo_progress_done "Archive downloaded"
+
+    echo_progress_start "Extracting archive"
+    tar xfv "/tmp/$app_name.tar.gz" --directory /opt/ >> "$log" 2>&1 || {
+        echo_error "Failed to extract"
+        exit 1
+    }
+    rm -rf "/tmp/$app_name.tar.gz"
+    chown -R "${user}": "$app_dir"
+    echo_progress_done "Archive extracted"
+}
+_install_prowlarr
+_systemd_prowlarr() {
```

Comment:
https://swizzin.ltd/dev/structure#install

function calls at the bottoms please